### PR TITLE
#60 adds translation for imprint

### PR DESCRIPTION
**Pull request by LeoTuet**
Translation for the `privacy statement` is still missing because the format of the text isn't that suitable for translation.
Should we remove the formating to make it more similar to the `imprint` and also to make it less complex to translate ?

The English translation is completly from DeepL since I was to lazy to translate it on my own. So if you find anything fishy let me know

### CODE REVIEW

**Comment by Aloso** [comment]
I think that Markdown would be a much more suitable format for pages like this. I found a [vite plugin](https://www.npmjs.com/package/vite-plugin-markdown) that can parse a Markdown file and create a react component for it. With this, we could have a imprint and privacy statement as 1 Markdown file for each language.

File: src/pages/PrivacyStatement.tsx [hunk 1]
```diff
@@ -3,713 +3,220 @@ import styles from "./ImprintAndPrivacy.module.scss";
...
+              <li><a href="#mOverview"> {t("privacy.overviewOverProcesses.header")} </a></li>
+              <li><a href="#mLegalBases"> {t("privacy.relevantLegalBases.header")} </a></li>
+              <li><a href="#mSecurity"> {t("privacy.security.header")} </a></li>
+              <li><a href="#mDeletion"> {t("privacy.deletionOfData.header")} </a></li>
+              <li><a href="#mCookies"> {t("privacy.useOfCookies.header")} </a></li>
+              <li><a href="#mWebhosting"> {t("privacy.webhosting.header")} </a></li>
+              <li><a  href="#mVote"> {t("privacy.votes.header")} </a></li>
+              <li><a href="#mChanges"> {t("privacy.updates.header")} </a></li>
+              <li><a href="#mRights"> {t("privacy.rights.header")} </a></li>
+              <li><a href="#mDefinitions"> {t("privacy.definitions.header")} </a></li>
```
Comment (94726) [inline src/pages/PrivacyStatement.tsx:34]: why `m`?

**Review by 94726 — CHANGES_REQUESTED** [review]

File: src/pages/PrivacyStatement.tsx (hunk 1 above)
Comment (94726) [reply src/pages/PrivacyStatement.tsx:34]: Also, I think it is uncommon to have any upper cased letters in urls as those fragment links are case sensitive.

Comment (LeoTuet) [reply src/pages/PrivacyStatement.tsx:34]: Yeah I agree. I forgot that it is an url

Comment (Aloso) [reply src/pages/PrivacyStatement.tsx:34]: > I think it is uncommon to have any upper cased letters in urls as those fragment links are case sensitive.

The issue is that URL fragments are case *in*sensitive in Internet Explorer (but they aren't in other browsers).

File: src/pages/PrivacyStatement.tsx [hunk 2]
```diff
@@ -13,28 +13,28 @@ export const PrivacyStatement = () => {
...
-              <li><a href="#mDefinitions"> {t("privacy.definitions.header")} </a></li>
+              <li><a href="#intro"> {t("privacy.intro.header")} </a></li>
+              <li><a href="#responsible"> {t("privacy.responsibilities.header")} </a></li>
+              <li><a href="#overview"> {t("privacy.overviewOverProcesses.header")} </a></li>
+              <li><a href="#legal-bases"> {t("privacy.relevantLegalBases.header")} </a></li>
+              <li><a href="#security"> {t("privacy.security.header")} </a></li>
+              <li><a href="#deletion"> {t("privacy.deletionOfData.header")} </a></li>
+              <li><a href="#cookies"> {t("privacy.useOfCookies.header")} </a></li>
+              <li><a href="#webhosting"> {t("privacy.webhosting.header")} </a></li>
+              <li><a  href="#vote"> {t("privacy.votes.header")} </a></li>
```
Comment (94726) [inline src/pages/PrivacyStatement.tsx:31]: Is it just a visual error or are there two spaces behind the anchor tag?

Comment (LeoTuet) [reply src/pages/PrivacyStatement.tsx:31]: nah it isn't I fucked that up

**Review by 94726 — APPROVED** [review]

**Pull request merged**